### ISSUE TITLE
Enable the option to use the property raw name as a label

### ISSUE THREAD

**Issue by ljmotta** [vazco/uniforms#932]
Hi,

It's not possible to change how the label is made, only if I make my own JSONSchemaBridge. In my case, it's possible to have more words or one word that is entirely capitalized, or even symbols.

Example:
```
{
    title: 'Example',
    type: 'object',
    properties: {
      "input A": { type: 'string' },
      "PIN": { type: 'string' },
      "Test-X": { type: 'string' },
    },
    required: ['input A', 'input B'],
  };
```

This renders a form with the labels:
 - "Input a"
 - "Pin"
 - "Test x"

This happens on these lines:

https://github.com/vazco/uniforms/blob/acefaee45918c1db6589053859e142ec647d143c/packages/uniforms-bridge-json-schema/src/JSONSchemaBridge.ts#L264-L268

https://github.com/vazco/uniforms/blob/acefaee45918c1db6589053859e142ec647d143c/packages/uniforms-bridge-json-schema/src/JSONSchemaBridge.ts#L68-L70

Proposition:
I think this could be useful to more people, so I'd suggest to modify the `label: boolean` props on the `BaseForm` component adding the possibilites to control with the user wants to use the "toHumanLabel" function, the raw name or no label at all. (something like `label: "raw" | "none" | "auto"`).

**Comment by radekmie** [comment]
Hi @ljmotta. If you'd like to do it automatically, for all the fields, then yes, you'd need a custom bridge. However, this custom bridge is fairly simple:
```ts
class CustomJSONSchemaBridge extends JSONSchemaBridge {
  getProps(name, props) {
    const finalProps = super.getProps(name, props);
    if (finalProps.label) finalProps.label = name;
    return finalProps;
  }
}
```
_(I haven't tested it.)_

If you can alter your schemas, i.e., preprocess them before using them in uniforms, then add a [`title` property](https://json-schema.org/understanding-json-schema/reference/generic.html), as the [bridge will use it](https://github.com/vazco/uniforms/blob/698187ed25c04df86971d712d71d231fa68c190e/packages/uniforms-bridge-json-schema/src/JSONSchemaBridge.ts#L264-L268).

And finally, about the `label` prop. It can't be incorporated into this single-prop easily, as it already serves a different purpose -configuring the flow of displaying labels (`true`/`false`/`null`/`string`). That'd mean a `label: 'raw'` would block other fields from being named `raw` and automatically capitalizing its name.

**Comment by ljmotta** [comment]
Hi @radekmie, indeed I could extend the actual `JSONSchmeaBridge`! I didn't thought about that for this problem. But after trying this solution, I realized that it has another problem. My project is targeting `es5`, and the `JSONSchemaBridge/es5` uses the `Bridge/es6`, so it's not possible to extend the JSONSchemaBridge and avoid the `cannot be invoked without 'new'` error (or probably I've done something wrong?). Even after changing the target of my project to check this solution, something wrong happened and the form didn't rendered haha. So I sticked with the preprocessing option and it worked fine. Didn't know about the `title` property! Thanks
